variables.tf

```diff
@@ -197,8 +203,9 @@ variable "min_size" {
 }
 
 variable "subnet_ids" {
-  description = "A list of subnet IDs to launch resources in"
   type        = list(string)
+  description = "A list of subnet IDs to launch resources in"
+  default     = null
```

Comment:
I think we want to remove this since this variable is still required.

Reply:
```suggestion
```

Reply:
Shoot. Missed that on my previous commit. Thanks for spotting it, I've removed the default value.